git-server/src/lib.rs

```diff
@@ -305,54 +318,68 @@ pub async fn run(options: Options) -> anyhow::Result<()> {
         bail!("Failed to disable gc: {:?}", e);
     }
 
-    let peer_id = path::param::<PeerId>()
-        .map(Some)
-        .or_else(|_| async { Ok::<(Option<PeerId>,), Infallible>((None,)) });
-
-    let server = warp::filters::any::any()
-        .map(move || ctx.clone())
-        .and(warp::method())
-        .and(warp::filters::header::headers_cloned())
-        .and(warp::filters::body::aggregate())
-        .and(warp::filters::addr::remote())
-        .and(path::param())
-        .and(peer_id)
-        .and(path::peek())
-        .and(
-            warp::filters::query::raw()
-                .or(warp::any().map(String::default))
-                .unify(),
+    let app = git_handler
+        .layer(Extension(ctx.clone()))
+        .layer(
+            TraceLayer::new_for_http()
+                .make_span_with(DefaultMakeSpan::new().level(Level::INFO))
```

Comment:
I opted for response only logging, with default level set to `INFO`. Logging now looks like:

```
Aug 18 19:45:05.308  INFO radicle_git_server: version 0.1.0-7c60de6
Aug 18 19:45:05.309  INFO radicle_git_server: git version 2.34.1
Aug 18 19:45:05.309  INFO shared: Profile 7a84311f-a61b-4700-8e3f-0ff3fe65eeee loaded...
Aug 18 19:45:05.363  INFO radicle_git_server: alias "nakamoto" for "rad:git:hnrkgswoskaqzdc1b3f8trncgaw736nfbjxdy"
Aug 18 19:45:05.365  INFO radicle_git_server: listening on http://0.0.0.0:8778
Aug 18 19:45:08.042  INFO request{method=GET uri=/nakamoto.git/info/refs?service=git-upload-pack version=HTTP/1.1}: radicle_git_server: git-http-backend: exited successfully for rad:git:hnrkgswoskaqzdc1b3f8trncgaw736nfbjxdy
Aug 18 19:45:08.043  INFO request{method=GET uri=/nakamoto.git/info/refs?service=git-upload-pack version=HTTP/1.1}: radicle_git_server: STATUS=200 LATENCY=11.613317ms
Aug 18 19:45:08.060  INFO request{method=POST uri=/nakamoto.git/git-upload-pack version=HTTP/1.1}: radicle_git_server: git-http-backend: exited successfully for rad:git:hnrkgswoskaqzdc1b3f8trncgaw736nfbjxdy
Aug 18 19:45:08.061  INFO request{method=POST uri=/nakamoto.git/git-upload-pack version=HTTP/1.1}: radicle_git_server: STATUS=200 LATENCY=13.540688ms
```

A bit long, but now there's only one line per hit. The other lines are warnings/logs/etc.

Reply:
@cloudhead do you prefer it this way? Cause I have to change https://github.com/radicle-dev/radicle-client-services/pull/177 as well.

Reply:
Responded in the other PR, I'm not sure what "response only logging" means..